app/src/features/surveys/view/survey-animals/form-sections/MeasurementAnimalForm.tsx

```diff
@@ -160,46 +106,46 @@ const MeasurementFormContent = ({ index, measurements }: MeasurementFormContentP
             route="xref/taxon-qualitative-measurement-options"
             query={`taxon_measurement_id=${taxonMeasurementId}`}
             controlProps={{
-              size: 'small',
               required: true,
               disabled: !taxonMeasurementId
             }}
+            handleChangeSideEffect={handleQualOptionName}
           />
         ) : (
           <Field
             as={CustomTextField}
             name={valueName}
             handleBlur={handleBlur}
-            label={measurement?.unit ? `Value [${measurement.unit}'s]` : `Value`}
+            label={currentMeasurement?.unit ? `Value [${currentMeasurement.unit}'s]` : `Value`}
```

Comment:
Same comment here re: adding an `'s` to the units.

Reply:
Updated